smoke/publish-local-registry.sh

```
@@ -1,6 +1,90 @@
 #!/bin/sh -e
 set -e
 
+# Function to verify package is published and available
+verify_package_published() {
+  local package_name=$1
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)